_posts/documentation/learn/2000-01-05-page-automation.md

```diff
@@ -11,7 +11,7 @@ Because PhantomJS can load and manipulate a web page, it is perfect to carry out
 
 Since the script is executed as if it is running on a web browser, standard [DOM scripting](http://en.wikipedia.org/wiki/DOM_scripting) and [CSS selectors](http://www.w3.org/TR/css3-selectors/) work just fine.
 
-The following `useragent.js` example demonstrates reading the `textContent` property of the element whose *id* is `myagent`:
+The following `useragent.js` example demonstrates reading the `textContent` property of the element whose *id* is `qua`:
```

Comment:
What is `qua` here? please explain.

Reply:
@gidcs if so, whats the whole point pf having it named differently? It only substitues `user-agent`, and introduces mess. Iam against it.

Reply:
The website changed the id which stored the actual element.
I just want to fix the broken code.
If you think the broken code is fine, it is okay.